tools/qt5_to_qt6_country_code_table.py

```diff
@@ -40,23 +40,23 @@ def parse_enum_from_header(header):
             # resolve aliases:
             name_to_val[name] = name_to_val[val]
         else:
-            raise("Unhandled case")
+            raise ValueError("Unhandled case")
     return name_to_val, val_to_name
 
 
 def make_struct(name, table):
     ret = ''
-    highest_key = max([int(x) for x in table.keys()])
-    ret += 'constexpr int const static %s[] = {' % (name,)
-    for key in range(highest_key+1):
-        ret += '%d' % table.get(key, -1)  # fill gaps with -1
+    highest_key = max(int(x) for x in table.keys())
+    ret += f"constexpr int const static {name}[]"
+    for key in range(highest_key + 1):
```

Comment:
One off error?
--> semantic check.

Reply:
Yes I wondered about this code when it came up, but didn't want to start mixing "fixes" in with this linting PR really

Reply:
Please LMK what you'd like done!

Reply:
Probably a verification that it does what it should do.

Reply:
Have tried running (and fixed my mistake in the above). Seems to work fine. 

Note that range is exclusive at the top end, so to include the highest key actually found we _do_ need to add one. 0 seems to be valid too:
```python
>>> print(list(range(3)))
[0, 1, 2]
```

Also a similar before vs after test: 
```shell
$ tools/qt5-to-qt6-country-code-table.py | md5sum
6b286414415e31e6864f74b0f73551af  -
```
vs new 
```shell
$ tools/qt5_to_qt6_country_code_table.py | md5sum
6b286414415e31e6864f74b0f73551af  -
```

Reply:
Will need to think about this a bit more. I think this piece of code in general needs verification...

Reply:
OK. For now it's behaving as before on my system.

Reply:
Yes, local testing gives the same output. It seems to be ok.